core/helper_api.php

```diff
@@ -89,13 +89,11 @@ function helper_alternate_class( $p_index = null, $p_odd_class = 'row-1', $p_eve
  * @return array|mixed transposed array or $p_array if not 2-dimensional array
  */
 function helper_array_transpose( array $p_array ) {
-	if( !is_array( $p_array ) ) {
```

Comment:
It is not necessary - thanks to the `array` type hint $p_array MUST be an array.

http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration